views/pandigital.erb

```diff
@@ -0,0 +1,25 @@
+<div class="problemdiv">
+  <h1>Pandigital Products</h1>
+  <br />
```

Comment:
Try to use paragraphs correctly, rather than relying on `<br>` tags for spacing.